configure.ac

```diff
@@ -197,6 +197,12 @@ GOBJECT_INTROSPECTION_CHECK([1.31.1])
 AM_CONDITIONAL([HAVE_INTROSPECTION], [test "$enable_introspection" = "yes"])
 
 # ------------------------------------------------------------------------------
+AC_ARG_ENABLE([programs],
+        AS_HELP_STRING([--disable-programs], [disable programs (udevd, udevadm and helpers)]),
+        [], [enable_programs="yes"])
```

Comment:
Please consider what happens if the user supplies `--enable-programs` to configure.

AFAIU, the 4th (empty) argument to AC_ARG_ENABLE will execute, and the below test will not be true.